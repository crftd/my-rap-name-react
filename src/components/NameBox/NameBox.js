import React, { Component } from 'react';

class NameBox extends Component {
  render() {
    return (
      <div>Lil Pimp</div>
    );
  }
}

export default NameBox;
