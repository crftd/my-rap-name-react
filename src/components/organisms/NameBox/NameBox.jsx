import React from 'react';
import { Chip } from '@material-ui/core';
import Avatar from '@material-ui/core/Avatar';
import styled from 'styled-components';

import rapper from '../../../assets/rapper.svg';

const NameBoxWrapper = styled.div`
  padding-top: 16px;
  display: flex;
  justify-content: center;
`;

export default ({ rapName }) => (
  <NameBoxWrapper>
    <Chip
      style={{ height: 'auto' }}
      icon={
        <Avatar
          style={{
            marginTop: 2,
            marginBottom: 2,
            maxHeight: 32,
            maxWidth: 32,
          }}
          src={rapper}
          alt="Rapper image"
        />
      }
      label={rapName}
      color="secondary"
    />
  </NameBoxWrapper>
);
