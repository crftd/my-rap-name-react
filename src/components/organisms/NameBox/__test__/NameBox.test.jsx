import React from 'react';
import { shallow } from 'enzyme';
import NameBox from '../NameBox.jsx';

describe('[Snapshot] NameBox', () => {
  it('should render', () => {
    // Arrange

    // Act
    const actualTree = shallow(<NameBox />);

    // Assert
    expect(actualTree).toMatchSnapshot();
  });
});
