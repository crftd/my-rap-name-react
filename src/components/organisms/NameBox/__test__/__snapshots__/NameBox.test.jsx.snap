// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Snapshot] NameBox should render 1`] = `
<styled.div>
  <WithStyles(Chip)
    color="secondary"
    icon={
      <WithStyles(Avatar)
        alt="Rapper image"
        src="rapper.svg"
        style={
          Object {
            "marginBottom": 2,
            "marginTop": 2,
            "maxHeight": 32,
            "maxWidth": 32,
          }
        }
      />
    }
    style={
      Object {
        "height": "auto",
      }
    }
  />
</styled.div>
`;
